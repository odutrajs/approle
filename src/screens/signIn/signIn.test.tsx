import React from "react";
import { renderWithProviders } from "../../../jest.setup";
import SignInScreen from "./index";

describe("SignInScreen", () => {
  it("should render correctly", () => {
    const tree = renderWithProviders(<SignInScreen />);
    expect(tree).toMatchSnapshot();
  });
});
