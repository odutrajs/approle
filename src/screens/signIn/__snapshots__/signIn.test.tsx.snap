// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SignInScreen should render correctly 1`] = `
<SafeAreaProvider>
  <ForwardRef(NavigationContainerInner)>
    <ThemeProvider
      theme={
        {
          "borderRadii": {
            "none": 0,
            "rounded": 4,
            "rounded-2xl": 10,
            "rounded-3xl": 12,
            "rounded-4xl": 16,
            "rounded-5xl": 20,
            "rounded-7xl": 28,
            "rounded-xl": 8,
          },
          "colors": {
            "amber100": "#fef3c7",
            "amber200": "#fde68a",
            "amber300": "#fcd34d",
            "amber400": "#fbbf24",
            "amber50": "#fffbeb",
            "amber500": "#f59e0b",
            "amber600": "#d97706",
            "amber700": "#b45309",
            "amber800": "#92400e",
            "amber900": "#78350f",
            "blu100": "#dbeafe",
            "blu200": "#bfdbfe",
            "blu300": "#93c5fd",
            "blu400": "#60a5fa",
            "blu50": "#eff6ff",
            "blu500": "#007AFF",
            "blu600": "#2563eb",
            "blu700": "#1d4ed8",
            "blu800": "#1e40af",
            "blu900": "#1e3a8a",
            "fuchsia100": "#fae8ff",
            "fuchsia200": "#f5d0fe",
            "fuchsia300": "#f0abfc",
            "fuchsia400": "#e879f9",
            "fuchsia50": "#fdf4ff",
            "fuchsia500": "#d946ef",
            "fuchsia600": "#c026d3",
            "fuchsia700": "#a21caf",
            "fuchsia800": "#86198f",
            "fuchsia900": "#701a75",
            "gray100": "#f3f4f6",
            "gray200": "#e5e7eb",
            "gray250": "#efeff0",
            "gray300": "#d1d5db",
            "gray4": "#898681",
            "gray400": "#9ca3af",
            "gray5": "#887d8e",
            "gray500": "#6b7280",
            "gray550": "#939292",
            "gray600": "#4b5563",
            "gray650": "#8a8a8a",
            "gray700": "#374151",
            "gray8": "#29262E",
            "gray800": "#1f2937",
            "gray9": "#101928",
            "gray900": "#111827",
            "green100": "#dcfce7",
            "green200": "#bbf7d0",
            "green300": "#86efac",
            "green400": "#4ade80",
            "green50": "#f0fdf4",
            "green500": "#22c55e",
            "green600": "#16a34a",
            "green700": "#15803d",
            "green800": "#166534",
            "green900": "#14532d",
            "grey": "#878787",
            "lightGray": "#EDECEF",
            "neutral100": "#f5f5f5",
            "neutral200": "#e5e5e5",
            "neutral300": "#d4d4d4",
            "neutral400": "#a3a3a3",
            "neutral500": "#737373",
            "neutral600": "#525252",
            "neutral700": "#404040",
            "neutral800": "#262626",
            "neutral900": "#171717",
            "orange100": "#ffedd5",
            "orange200": "#fed7aa",
            "orange300": "#fdba74",
            "orange400": "#fb923c",
            "orange50": "#fff7ed",
            "orange500": "#f97316",
            "orange600": "#ea580c",
            "orange700": "#c2410c",
            "orange800": "#9a3412",
            "orange900": "#7c2d12",
            "pink100": "#fce7f3",
            "pink200": "#fbcfe8",
            "pink300": "#f9a8d4",
            "pink400": "#f472b6",
            "pink50": "#fdf2f8",
            "pink500": "#ec4899",
            "pink600": "#db2777",
            "pink700": "#be185d",
            "pink800": "#9d174d",
            "pink900": "#831843",
            "primary": "#fba72a",
            "purple100": "#f3e8ff",
            "purple200": "#e9d5ff",
            "purple300": "#d8b4fe",
            "purple400": "#c084fc",
            "purple50": "#faf5ff",
            "purple500": "#a855f7",
            "purple600": "#9333ea",
            "purple700": "#7e22ce",
            "purple800": "#6b21a8",
            "purple900": "#581c87",
            "red100": "#fee2e2",
            "red200": "#fecaca",
            "red300": "#fca5a5",
            "red400": "#f87171",
            "red50": "#fef2f2",
            "red500": "#ef4444",
            "red600": "#dc2626",
            "red700": "#b91c1c",
            "red800": "#991b1b",
            "red900": "#7f1d1d",
            "rose100": "#ffe4e6",
            "rose200": "#fecdd3",
            "rose300": "#fda4af",
            "rose400": "#fb7185",
            "rose50": "#fff1f2",
            "rose500": "#f43f5e",
            "rose600": "#e11d48",
            "rose700": "#be123c",
            "rose800": "#9f1239",
            "rose900": "#881337",
            "sky100": "#e0f2fe",
            "sky200": "#bae6fd",
            "sky300": "#7dd3fc",
            "sky400": "#38bdf8",
            "sky50": "#f0f9ff",
            "sky500": "#0ea5e9",
            "sky600": "#0284c7",
            "sky700": "#0369a1",
            "sky800": "#075985",
            "sky900": "#0c4a6e",
            "slate100": "#f1f5f9",
            "slate200": "#e2e8f0",
            "slate300": "#cbd5e1",
            "slate400": "#94a3b8",
            "slate500": "#64748b",
            "slate600": "#475569",
            "slate700": "#334155",
            "slate800": "#1e293b",
            "slate900": "#0f172a",
            "transparent": "transparent",
            "violet100": "#e0e7ff",
            "violet200": "#c7d2fe",
            "violet300": "#a5b4fc",
            "violet400": "#818cf8",
            "violet50": "#eef2ff",
            "violet500": "#6366f1",
            "violet600": "#4f46e5",
            "violet700": "#4338ca",
            "violet800": "#3730a3",
            "violet900": "#312e81",
            "white": "#ffffff",
            "zinc100": "#f4f4f5",
            "zinc200": "#e4e4e7",
            "zinc300": "#d4d4d8",
            "zinc400": "#a1a1aa",
            "zinc500": "#71717a",
            "zinc550": "#848488",
            "zinc600": "#52525b",
            "zinc700": "#3f3f46",
            "zinc800": "#27272a",
            "zinc900": "#18181b",
          },
          "default": {
            "fontFamily": "Gotham",
          },
          "spacing": {
            "1": 4,
            "10": 40,
            "11": 44,
            "12": 48,
            "13": 56,
            "2": 8,
            "3": 12,
            "3.5": 14,
            "4": 16,
            "5": 20,
            "5.5": 22,
            "6": 24,
          },
          "textVariants": {
            "defaults": {},
            "text2Xl": {
              "color": "gray8",
              "fontSize": 24,
              "lineHeight": 32,
            },
            "text3Xl": {
              "color": "gray8",
              "fontSize": 36,
              "lineHeight": 40,
            },
            "textBase": {
              "color": "gray8",
              "fontSize": 16,
              "lineHeight": 24,
            },
            "textLg": {
              "color": "gray8",
              "fontSize": 18,
              "lineHeight": 28,
            },
            "textSm": {
              "color": "gray8",
              "fontSize": 14,
              "lineHeight": 18,
            },
            "textXl": {
              "color": "gray8",
              "fontSize": 20,
              "lineHeight": 28,
            },
            "textXs": {
              "color": "gray8",
              "fontSize": 12,
              "lineHeight": 16,
            },
          },
        }
      }
    >
      <SWRConfig
        value={
          {
            "provider": [Function],
          }
        }
      >
        <SignInScreen />
      </SWRConfig>
    </ThemeProvider>
  </ForwardRef(NavigationContainerInner)>
</SafeAreaProvider>
`;
